network/p2p/blob/blob_service.go

```diff
@@ -293,3 +294,57 @@ func AuthorizedRequester(
 		return true
 	}
 }
+
+type Tracer struct {
+	logger zerolog.Logger
+}
+
+func NewTracer(logger zerolog.Logger) *Tracer {
+	return &Tracer{
+		logger,
+	}
+}
+
+func (t *Tracer) logMsg(msg bsmsg.BitSwapMessage, s string) {
+	evt := t.logger.Debug()
+
+	wantlist := zerolog.Arr()
+	for _, entry := range msg.Wantlist() {
+		wantlist = wantlist.Interface(entry)
```

Comment:
What other fields will this print besides the CID?

Reply:
https://pkg.go.dev/github.com/ipfs/go-bitswap@v0.10.2/message#Entry